paddle/scripts/travis/build_doc.sh

```diff
@@ -1,15 +1,18 @@
 #!/bin/bash
+set -e
+
+# Create the build directory for CMake.
+mkdir -p $TRAVIS_BUILD_DIR/build
+cd $TRAVIS_BUILD_DIR/build
 
-# Add set -e, cd to directory.
-source ./common.sh
 # Compile Documentation only.
-cmake .. -DCMAKE_BUILD_TYPE=Debug -DCMAKE_Fortran_COMPILER=/usr/bin/gfortran-4.8 -DWITH_GPU=OFF -DWITH_DOC=OFF -DWITH_STYLE_CHECK=OFF ${EXTRA_CMAKE_OPTS}
+cmake .. -DCMAKE_BUILD_TYPE=Debug -DWITH_GPU=OFF -DWITH_DOC=OFF -DWITH_STYLE_CHECK=OFF
 mkdir output
 make -j `nproc`
```

Comment:
Because part of Paddle's Python code is generated by C++ header.  Only after we complete compiling PaddlePaddle, Paddle's Python package can be built. And `Sphinx` can load this python package to generate documentation.

So, generate documentation needs Python package. Python package needs to compile Paddle.